theia-full-docker/Dockerfile

```diff
@@ -176,18 +183,47 @@ RUN apt-get update && apt-get -y install openjdk-11-jdk maven gradle
 
 # C/C++
 # public LLVM PPA, development version of LLVM
-ARG LLVM=13
-
-RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
-    echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main" > /etc/apt/sources.list.d/llvm.list && \
-    apt-get update && \
-    apt-get install -y \
-                       clang-tools-$LLVM \
-                       clangd-$LLVM \
-                       clang-tidy-$LLVM \
-                       gcc-multilib \
-                       g++-multilib \
-                       gdb && \
+# depending on architecture install form public LLVM PPA (Verison 13) or distirbution (Version 11)
+ARG LLVM=11
```

Comment:
@dm4tze I believe the downgrade to `11` is problematic, we want to use the latest version.

cc @marcdumais-work

Reply:
I'm not an LLVM or C/C++ Compiler expert, so please be kind if I miss something.
For me, it looks like 11 is the latest stable version and 12 will be released next month (see https://llvm.org/).
Version 13 seems to be the snapshot version.

At the moment I will install version 13 for amd64 build and version 11 for arm64 build. (I will make that more clear in the Dokerfile)

If you insist on installing LLVM Version 13, the only solution I see at the moment is installing it from source. Would this be a feasible solution for you?

Thank you for your support!

Reply:
@dm4tze I think that's fine, if `13` is not available for your architecture (arm64) then we can use a version which is supported in that use-case.

Reply:
I want to add that the reason we want to bundle the latest development `clangd` and related tool is that we indirectly participate in its development and want to showcase the latest and greatest version. v11 will still work fine if nothing else is available, with some reduced features.

Reply:
If you have connections to the LLVM team maybe you could ask them if they would publish the arm64 packages on their repository.

Until then, I have set the default value to 13 and only downgrade on arm build.